File: plugin-tester-scala/src/main/scala/io/grpc/examples/helloworld/GreeterImpl.scala
package io.grpc.examples.helloworld

import scala.concurrent.Future

import akka.NotUsed
import akka.stream.scaladsl.Source

class GreeterImpl extends Greeter {
  override def sayHello(in: HelloRequest): Future[HelloReply] = ???

  override def streamHellos(in: Source[HelloRequest, _]): Source[HelloReply, Any] = ???

  override def itKeepsTalking(in: Source[HelloRequest,_]): Future[HelloReply] = ???
}

File: plugin-tester-java/src/main/java/io/grpc/examples/GreeterImpl.java
package io.grpc.examples;

import java.util.concurrent.CompletableFuture;

import akka.stream.javadsl.Source;

public class GreeterImpl implements Greeter {
  @Override
  public CompletableFuture<io.grpc.examples.HelloReply> sayHello(io.grpc.examples.HelloRequest in) {
    throw new UnsupportedOperationException();
  }

  @Override
  public CompletableFuture<io.grpc.examples.HelloReply> itKeepsTalking(Source<io.grpc.examples.HelloRequest, Object> in) {
    throw new UnsupportedOperationException();
  }

  @Override
  public Source<io.grpc.examples.HelloReply, Object> streamHellos(Source<io.grpc.examples.HelloRequest, Object> in) {
    throw new UnsupportedOperationException();
  }
}
